jax/_src/numpy/fft.py

```diff
@@ -109,10 +109,76 @@ def _fft_core(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return transformed
 
 
-@implements(np.fft.fftn)
 def fftn(a: ArrayLike, s: Shape | None = None,
          axes: Sequence[int] | None = None,
          norm: str | None = None) -> Array:
+  r"""Compute the multidimensional discrete fourier transform.
+
+  JAX implementation of :func:`numpy.fft.fftn`.
+
+  Args:
+    a: input array
+    s: integer or sequence of integers. Specifies the shape of the result. If not
+      specified, it will default to the shape of ``a`` along the specified ``axes``.
+    axes: integer or sequence of integers, default=None. Specifies the axes along
+      which the transform is computed.
+    norm: string. The normalization mode. "backward", "ortho" and "forward" are
+      supported.
```

Comment:
It would be nice to describe what the different normalizations mean somewhere. We don't currently and I don't think it needs to be part of this PR, but I wanted to note that point while I remembered it!